content/en/dashboards/_index.md

```diff
@@ -52,34 +40,43 @@ cascade:
 
 ## Overview
 
-A dashboard is Datadog's tool for visually tracking, analyzing, and displaying key performance metrics, which enable you to monitor the health of your infrastructure.
-
-**Note**: View Dashboards with the [Datadog Mobile App][1], available on the [Apple App Store][2] and [Google Play Store][3].
+Dashboards provide real-time insights into the performance and health of systems and applications within an organization. They allow users to visually analyze data, track key performance indicators (KPIs), and monitor trends efficiently. With dashboards, teams can identify anomalies, prioritize issues, proactively detect problems, diagnose root causes, and ensure that reliability goals are met. Empower your teams to make informed decisions, optimize system operations, and drive business success by providing a centralized and user-friendly interface for monitoring and analyzing critical metrics and performance indicators.
 
 {{< whatsnext desc="Dashboard features:">}}
     {{< nextlink href="/dashboards/configure" >}}Configure: Overview of the configuration options for dashboards{{< /nextlink >}}
+    {{< nextlink href="/dashboards/configure" >}}Dashboard List: Search, view, or create dashboards and lists{{< /nextlink >}}
+    {{< nextlink href="/dashboards/template_variables" >}}Template Variable: Dynamically filter widgets in a dashboard{{< /nextlink >}}
+    {{< nextlink href="/service_management/incident_management/datadog_clipboard/" >}}Datadog Clipboard{{< /nextlink >}}
+    {{< nextlink href="/api/latest/dashboards" >}}API: Manage dashboards programmatically{{< /nextlink >}}
+{{< /whatsnext >}}
+
+{{< whatsnext desc="Graphing features:">}}
     {{< nextlink href="/dashboards/widgets" >}}Widgets: Learn the configuration for different visualizations{{< /nextlink >}}
     {{< nextlink href="/dashboards/querying" >}}Querying: See the formatting options for graph queries{{< /nextlink >}}
     {{< nextlink href="/dashboards/functions" >}}Functions: Modify metric queries and resulting graphs{{< /nextlink >}}
-    {{< nextlink href="/dashboards/template_variables" >}}Template Variable: Dynamically filter widgets in a dashboard{{< /nextlink >}}
     {{< nextlink href="/dashboards/change_overlays" >}}Overlays: Automatically overlay change events on graphs{{< /nextlink >}}
-    {{< nextlink href="/api/latest/dashboards" >}}API: Manage dashboards programmatically{{< /nextlink >}}
 {{< /whatsnext >}}
 
-## New dashboard
+## Get started
+
+{{< whatsnext desc="See the following resources to get started with Dashboards:" >}}
```

Comment:
```suggestion
{{< whatsnext desc="See the following resources:" >}}
```
Just because the repetition here with the first item in the list feels a little odd, especially when the heading is also "Get started".